config/vendor-api-v1.yml

```diff
@@ -429,6 +429,7 @@ components:
             lifecycle diagram](/api-docs/lifecycle) for states and transitions.
           enum:
           - awaiting_provider_decision
+          - interviewing
```

Comment:
We can't add this as it's a breaking change

Reply:
Why is that? Do we need to update the description then, as the lifecycle diagram does render the interviewing state (which is why I made the change here)?

Reply:
SRSs won't know what to do with this state if they receive it over the API — obv we haven't started sending it here but it could still cause alarm for those with integrations already live or in late stages of development. We should probably tackle the lifecycle diagram. I think we can knock states out of it fairly easily.

Reply:
@duncanjbrown ok updated the PR. We can add a different card for the lifecycle diagram.